### ISSUE TITLE
[Bug] Wrong video size

### ISSUE THREAD

**Issue by Aliaksandr-Kasko-JazzTeam** [Flam3rboy/puppeteer-stream#15]
Even when I create browser with defined viewport (i.e. 300*600)
```
const width = 300;
const height = 600;

const browser = await launch({
        defaultViewport: {
          width: width,
          height: height
        },
        args: ['--no-sandbox']
      });
//...
```
video size of recorded video will be 1920*1080.
What am I doing wrong?

**Comment by KennyHarrer** [comment]
Hello Aliaksandr! Try adding `'--window-size=1920,1080', '--start-fullscreen'` to your args.

**Comment by Aliaksandr-Kasko-JazzTeam** [comment]
Hi, @KennyHarrer.
No, it doesn't matter.
Even when I set ```args: ['--no-sandbox', `--window-size=${width},${height}`, '--start-fullscreen']``` or without `'--start-fullscreen'`. All the same. Video size is 1920*1080. I need it to be 300*600 in this case

**Comment by KennyHarrer** [comment]
Have you tried setting the page view port?
`page.setViewport({width:300,height:600})`

**Comment by Aliaksandr-Kasko-JazzTeam** [comment]
Yes, I tried this. And it has no effect.

**Comment by KennyHarrer** [comment]
I am now also having this issue.

**Comment by InsideBSITheSecond** [comment]
I fixed this by piping it to a ffmpeg instead of an fs stream
You do that by doing (this is assuming an audio stream)
(and you need to have ffmpeg binaries installed)
```
const { exec } = require("child_process");
ffmpeg = exec(`ffmpeg -y -i - output.wav`);
stream.pipe(ffmpeg.stdin);
```

**Comment by StuartGooding** [comment]
I believe that the background extension that is invoking the chrome.tabCapture.capture needs to allow for videoConstraints to be passed through to override the minWidth, maxWidth, minHeight and maxHeight of the video. I've found this can allow for the output video size to match your expected resolution.

**Comment by Flam3rboy** [comment]
resolved with #32
